libs/blocks/editorial-card/editorial-card.css

```
@@ -224,11 +224,20 @@
 }
 
 .editorial-card.media-tall .media-area img,
-.editorial-card.media-tall .media-area video {
+.editorial-card.media-tall .media-area video,
+.editorial-card.media-tall .media-area .milo-video {
   aspect-ratio: var(--aspect-ratio-tall);
   max-height: unset;
 }
 
+.editorial-card.media-square .media-area .milo-video,
+.editorial-card.media-wide .media-area .milo-video,
+.editorial-card.media-standard .media-area .milo-video,
+.editorial-card.media-tall .media-area .milo-video {
```

Maybe some `:is` would keep this a bit more succint:
```suggestion
.editorial-card:is(.media-square, .media-wide, .media-standard, .media-tall) .media-area .milo-video
```